OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -512,7 +517,8 @@ protected override MovePart OnComplete(Actor self, Mobile mobile, Move parent)
 							ToTerrainOrientation,
 							nextToTerrainOrientation,
 							margin,
-							progress - Distance);
+							progress - Distance,
+							mobile.ToCell.Layer == 0 && nextCell.Value.Cell.Layer == 0);
```

Comment:
For my own understanding, we have `FromCell`, `ToCell`, `nextCell` in scope here. Why are only the latter two important for this check?

Reply:
See `.SetLocation` a few lines below. We care about how we are going to move into the next cell

Reply:
FromCell is redundant here, as we have already passed it